Exercises/01/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.16)
+project(KokkosTutorial01)
+
+add_subdirectory(Begin)
+add_subdirectory(Solution)
```

Comment:
Hmm how about switching to two source files and two targets in one directory for per exercise?
This would drastically reduce repetition here and we could just tell participants to build different targets for begin and solution

Reply:
Not relevant for this PR. Sorry for the confusion. It was previously done on top of #91 to test easily if it was working.